app/connectors/BusinessDetailsConnector.scala

```diff
@@ -37,7 +38,7 @@ class GetBusinessDetailsConnector @Inject()(val http: HttpClient,
     }
   }
 
-  def headers: Seq[(String, String)] = appConfig.getIFHeaders("1171")
+  def headers: Seq[(String, String)] = Seq(("key", "value"))
```

Comment:
I am quite sure that we would need original values to be populated from appConfig.getIFHeaders("1171")